src/createjson/createJson.js

```diff
@@ -59,22 +62,45 @@ const createJson = () => {
 async function start(githubUsername) {
   await checkUser(githubUsername).then((result) => {
     if (result === true) {
-      questions().then((answers) => {
+      questions().then(async (answers) => {
         json = answers;
-        prompt([
+        await prompt([
           {
             type: "confirm",
             name: "addLink",
             message: "Do you want to add a link?",
           },
-        ]).then((answers) => {
-          const { addLink } = answers;
-          if (addLink) {
-            addlinkstojson();
-          } else {
-            createUser(githubUsername, json);
+        ]).then(async (answers) => {
+          if (answers.addLink) {
+            json.links = await addlinks(true);
+          }
+        });
+
+        await prompt([
+          {
+            type: "confirm",
+            name: "addMilestone",
+            message: "Do you want to add a milestone?",
+          },
+        ]).then(async (answers) => {
+          if (answers.addMilestone) {
+            json.milestones = await addmilestones(true);
           }
         });
+
+        await prompt([
```

Comment:
I think we should keep the milestone addition as a separate thing from CLI. Reason (main prompt). 

- When creating a profile it will too much to ask and people can also rush while writing it
- It will block merging into the main as it's not ready. Also, we need to do a little more on this because for adding testimonials to other people we need to create folders and then add JSON files.

Reply:
So I should remove the milestones for now ?

Reply:
No, I mean keep at seprate. As a separate option. WDYT? 
Also, do you agree with the above? 

> Yes, I saw that. I think you didn't get my point. If a person has no linkfree profile and they are creating a new one. How they have already got some testimonials? I think I am not explaining well 😅

Feel free to put your point

Reply:
> No, I mean to keep at separate option. WDYT? 
> Also, do you agree with the above? 
> 
> > Yes, I saw that. I think you didn't get my point. If a person has no linkfree profile and they are creating a new one. How they have already got some testimonials? I think I am not explaining well 😅
> 
> Feel free to put your point

Ok I will look into it